### DISCUSSION TITLE
How to use Nuscenes Dataparser to train the nerfacto Model

### DISCUSSION THREAD

**Discussion by mathiga** [nerfstudio-project/nerfstudio#3084] [Q&A]
HI ,

I am trying to understand the Nerfstudio, for that i am using the Nuscenes data set V1.o Mini , i have the datasets and used COLMAP to get needed information from the data sets for training the model, after read through the documentation, i understand we need to have a custom data parser, which will get he Raybundle from the image to model while training, I can see the Nuscenes dataparser and sample code of getting the dataparser output, but not sure how to use that and train the model

**Comment by MatteoMarengo** [comment]
Did you obtain a nice COLMAP reconstruction ?
